frontend/src/javascript/utils/codeUtils.js

```diff
@@ -0,0 +1,95 @@
+import store from "@/store";
+
+export function compareCode(oldText, newText, event, selection) {
+    let oldLen = oldText.length;
+    let newLen = newText.length;
+    let lengthDifference = newLen - oldLen;
+
+    let oldLineCount = oldText.numberOfLines();
+    let newLineCount = newText.numberOfLines();
+    let lineCountDifference = newLineCount - oldLineCount;
+
+    let ret = {
```

Comment:
ret to jest skrót od return? Często spotykałem się z konwencją nazewniczą, że wartość zwracaną nazywa się result i ja też w projekcie tej konwencji używałem.